src/bindings/python/PyBuiltinConfigRegistry.cpp

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright Contributors to the OpenColorIO Project.
+
+#include "PyBuiltinConfigRegistry.h"
+#include "PyUtils.h"
+
+namespace OCIO_NAMESPACE
+{
+
+enum BuiltinConfigRegistryIterator
+{
+    IT_BUILTIN_CONFIG_NAME = 0,
+    IT_BUILTIN_CONFIG
+};
+using BuiltinConfigNameIterator = PyIterator<PyBuiltinConfigRegistry, IT_BUILTIN_CONFIG_NAME>;
+using BuiltinConfigIterator = PyIterator<PyBuiltinConfigRegistry, IT_BUILTIN_CONFIG>;
+
+void bindPyBuiltinConfigRegistry(py::module & m)
+{
+    auto clsBuiltinConfigRegistry =
+        py::class_<PyBuiltinConfigRegistry>(m.attr("BuiltinConfigRegistry"));
+
+    auto clsBuiltinConfigNameIterator = 
+        py::class_<BuiltinConfigNameIterator>(
+            clsBuiltinConfigRegistry, "BuiltinConfigNameIterator");
+
+    auto clsBuiltinConfigIterator= 
+        py::class_<BuiltinConfigIterator>(
+            clsBuiltinConfigRegistry, "BuiltinConfigIterator");
+
+    clsBuiltinConfigRegistry
+        .def(py::init<>(), DOC(BuiltinConfigRegistry, Get))
+        .def("__iter__", [](PyBuiltinConfigRegistry & self)
+            { 
+                return BuiltinConfigNameIterator(self); 
+            },
+             DOC(BuiltinConfigRegistry, getBuiltinConfigName))
+        .def("__len__", [](PyBuiltinConfigRegistry & self) 
+            { 
+                return self.getNumBuiltinConfigs(); 
+            },
+            DOC(BuiltinConfigRegistry, getNumBuiltinConfigs))
+        .def("__getitem__", [](PyBuiltinConfigRegistry & self, const std::string & name) 
+            { 
+                return self.getBuiltinConfigByName(name.c_str());
+            },
+            DOC(BuiltinConfigRegistry, getBuiltinConfigByName))
+        .def("__contains__", [](PyBuiltinConfigRegistry & self, const std::string & name) 
+            {
+                const char * config = self.getBuiltinConfigByName(name.c_str());
+                if ((config != NULL) && (config[0] == '\0'))
+                {
+                    return false;
+                }
+                return true;
+            })
+        .def("getBuiltinConfigs", [](PyBuiltinConfigRegistry & self) 
+            {
+                return BuiltinConfigIterator(self);
+            })
+        .def("getNumBuiltinConfigs", [](PyBuiltinConfigRegistry & self) 
```

Comment:
[Minor] Could be removed now that we have the iterator support.

Reply:
Agreed. I'll remove it.